src/camvidlog2/cli.py

```
@@ -52,63 +58,67 @@ def load(
+        with open(outdir / "query.json", "w") as json_out:
+            json_out.write(embedding_group.model_dump_json(indent=2))
 
-    string_embeddings = get_string_embeddings(queries)
-    frame_embeddings = df.drop(columns=["filename", "frame_no"])
-
-    # dot product of unit vectors is the alignment between them (1 = equal, 0 = perpendicular)
-    distances = frame_embeddings.dot(string_embeddings.transpose())
-    # drop frame embedding columns and cleanup
-    df = df[["filename", "frame_no"]]
+    search_embeddings = load_embedding_group_dataframe(
+        embedding_group, video_embeddings
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)